synapse/_scripts/review_recent_signups.py

```diff
@@ -56,6 +58,9 @@ def get_recent_users(txn: LoggingTransaction, since_ms: int) -> List[UserInfo]:
             AND deactivated = 0
     """
 
+    if exclude_app_service:
+        sql += "\n AND appservice_id IS NOT NULL"
```

Comment:
```suggestion
        sql += " AND appservice_id IS NOT NULL"
```
There's nothing wrong with adding a newline, but I think we'd only add it for human readability.

Since humans are reading the source code, it would probably make more sense to not have the newline escape.